src/unstract/adapters/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.19.0"
+__version__ = "0.19.1"
```

Comment:
@gaya3-zipstack Also, since this is a patch version upgrade, do we need to update places refereed?

Reply:
We will pdm.lock file updates @harini-venkataraman

Reply:
We will pdm.lock file updates @harini-venkataraman